### PR TITLE
Addec Slides for C# Math

### PR DESCRIPTION
**Pull request by VishnuKulakkattu**
Kindly check and review

image

image

image

image

image

### CODE REVIEW

File: CSMathclass.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        <div class="slides">
+            <section>
+                <h2>C# MATH CLASS</h2>
+            </section>
+            <section>
+                <p id="centerpara">The C# Math class has many methods that allows you to perform mathematical tasks on numbers.</p>
+            </section>
+            <section>
+                <p id="centerpara">Self learn about:
+                    <a href="https://www.geeksforgeeks.org/c-sharp-math-class/">C# Math operators</a></p>
```
Comment (wandercode) [inline CSMathclass.html:29]: making it target blank may be good.

File: CSMathclass.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    <li>
+                        <p>Write a program to find the greater and smaller value of two variables( Compare 2 Int32 numbers, 2 decimal numbers, 2 byte numbers). </p>
+                    </li>
+                </ul>
+                <p>Expected output: </p><br><img src="/image/CSMathQ1.png" alt="">
+            </section>
+            <section>
+                <ul>
+                    <li>
+                        <p id="centerpara">Write a program to calculate the value that results from raising 3 to a power 6( Find 3 6 )?
```
Comment (wandercode) [inline CSMathclass.html:43]: change the 3 6 in bracket to 
image

**Review by wandercode — CHANGES_REQUESTED** [review]

File: CSMathclass.html (hunk 1 above)
Comment (VishnuKulakkattu) [reply CSMathclass.html:29]: Done

**Review by wandercode — APPROVED** [review]

**Comment by wandercode** [comment]
LGTM 👍

**Review by sneha-brainchild — APPROVED** [review]

**Pull request merged**